CONTRIBUTING.md

```diff
@@ -33,13 +33,40 @@ yarn build
 yarn build:watch
 ```
 
-## Run tests
+## Updating and adding packages
+
+Since this is a monorepo with several subrepos, if you want to update/add packages in the subrepos like `create-snowpack-app/app-template-11ty` you'll want to run the commands in the target subdirectory like
+
+```bash
+cd create-snowpack-app/app-template-vue
+yarn add vue@latest
+```
+
+## Tests
+
+Our test suite uses a mixture of unit, integration, and snapshot tests. We recommend running these before you submit a PR.
```

Comment:
Is "snapshot tests" a thing? I think we can maybe leave out this sentence, or shorten it to just 

```suggestion
We recommend running all tests before you submit a PR
```